cylc/uiserver/jupyterhub_config.py

```diff
@@ -49,21 +58,40 @@ def _load(path):
         exec(path.read_text())
 
 
-def load():
+def get_conf_dir_hierarchy(
+        config_paths: List[Path], filename: Optional[bool] = True
```

Comment:
`Optional[X]` means `Union[X, None]`. Not needed for default args unless the default is `None`

```suggestion
        config_paths: List[Path], filename: bool = True
```